less/layout.less

```diff
@@ -7,11 +7,22 @@ h1, h2, h3, h4, h5 {
     color: @secondary-color;
 }
 
+header .jumbotron {
+    background-color: @accent-color;
+}
+
+.nav navbar {
+    background-color: @accent-color;
+}
+
 .nav a {
     color: @secondary-color;
 }
 
-
+.main-content img {
```

Comment:
Your image seems to be slightly overlapping each other here, unless that was the way you intended it to be!